data/field/field.urdf

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" ?>
 <robot name="field.urdf">
     <link name="field_link">
+    <contact>
```

Comment:
Could you post a link for where you found documentation for these tags?

Reply:
I tried the <friction> and <ode> to the Field.urdf but it didn't work so I found the friction from a Japanese website. Here is the link :https://qiita.com/saketosakana/items/0e52997cbe43919ee2bd 
https://qiita.com/saketosakana/items/491d3e0cec0a0e288748
https://qiita.com/saketosakana/items/b4529df7ebc1f60356a6